report/41_Lessons-Evolution-&-Refactoring.md

```diff
@@ -78,8 +79,8 @@ It was then considered whether HTTPS and Traefik routing were the thing slowing
 The actual reason turned out to be the server location, as will be described in [TODO: 4.2 Operations](42_Lessons-Operation.md).
 
 Without the graph, the statistics, and the competitive nature of the team as motivators, these things would never have been discovered.
-Replication of this feeling would be difficult, as there rarely are 10 teams competing to implement the best and fastest project when looking at the job market.
-Gamification of collected metrics might be able to lead teams to improve these things, for example through having a big screen with several desired trend lines/thresholds along with actual statistics.
+Replication of this feeling later could be done by collecting metrics from competetitors, if there are any, and gamifying those metrics.
```

Comment:
> Replication of this feeling later could be done by collecting metrics from competetitors, if there are any, and gamifying those metrics.

Change to

"Replication of this feeling could be done by collecting metrics from competitors, and gamifying these."